plugin/action/mask/mask.go

```diff
@@ -148,26 +145,18 @@ func (p *Plugin) Start(config pipeline.AnyConfig, params *pipeline.ActionPluginP
 	p.valueNodes = make([]*insaneJSON.Node, 0)
 	p.logger = params.Logger
 	p.config.Masks = compileMasks(p.config.Masks, p.logger)
-	if p.config.MetricSubsystemName != nil {
```

Comment:
Metric registration is required only if `MetricSubsystemName` is declared in config. This `if` can't be removed.